client/signup/steps/import/util.ts

```diff
@@ -1,5 +1,7 @@
 import { ImporterPlatform } from './types';
 
+export const CAPTURE_URL_RGX = /^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-z0-9]+([-.][a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$/i;
```

Comment:
I've seen that in other parts of the code [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) is used to check if a URL is valid (not many though). So maybe in a future minor refactoring, we can substitute this regex.

Reply:
Agree. I'll update it in the subsequent iterations.